GameTests.java

```diff
@@ -30,8 +29,9 @@ public void playerMoveIsRegistered() {
     @Test
     public void multiplePlayerMovesAreRegistered() {
         InputStream stream = new ByteArrayInputStream("2\n3".getBytes());
+        BufferedReader inputReader = new BufferedReader(new InputStreamReader(stream));
```

Comment:
Just saw that ConsoleGame constructor takes a BufferedReader but Game takes the InputStream and then constructs a BufferedReader in it's method: _startNewGame()_.  
I think these constructors should be consistent with each other e.g. either both take InputReader or both BufferedReader.  Probably InputReader as it is higher level than BufferedReader.